python/lsst/ts/salobj/base.py

```diff
@@ -194,6 +194,9 @@ def astropy_time_from_tai_unix(tai_unix: float) -> astropy.time.Time:
         TAI time as unix seconds, e.g. the time returned by CLOCK_TAI
         on linux systems.
     """
+    warnings.warn(
+        "Use lsst.ts.utils.astropy_time_from_tai_unix instead", DeprecationWarning
+    )
     tai_mjd = (MJD_MINUS_UNIX_SECONDS + tai_unix) / SECONDS_PER_DAY
```

Comment:
Should you call the method from `utils` as well?

```
return utils.astropy_time_from_tai_unix(tais_unix=tais_unix)
```

Reply:
Good catch. Fixed. I also added a # TODO comment just before the function.